app/routes/task_routes.py

```diff
@@ -0,0 +1,116 @@
+from app import db
+from app.models.task import Task
+from .route_helpers import *
+from flask import Blueprint, request, make_response, jsonify
+import datetime
+
+bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@bp.route("", methods=["GET"])
+def get_tasks():
+    sort_query = request.args.get("sort")
+    search_title_query = request.args.get("search_title")
+    if sort_query:
+        if sort_query == "asc":
+            tasks = Task.query.order_by(Task.title).all()
```

Comment:
Rather than calling SqlAlchemy stuff directly from the routes, one thing we can consider  is to make class helper methods to represent the operations we expect to perform. This forms a more self-documenting interface about what operations we expect to perform. This also follows the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter).

We could make helper equivalents of things like `get`, `order_by`, `filter_by`, and `all`. Really, any of the `query` methods.